web/src/app/admin/users/page.tsx

```diff
@@ -8,30 +15,46 @@ import {
   TableBody,
   TableCell,
   Button,
+  Text,
 } from "@tremor/react";
 import { LoadingAnimation } from "@/components/Loading";
 import { AdminPageTitle } from "@/components/admin/Title";
-import { usePopup } from "@/components/admin/connectors/Popup";
+import { usePopup, PopupSpec } from "@/components/admin/connectors/Popup";
 import { UsersIcon } from "@/components/icons/icons";
 import { errorHandlingFetcher } from "@/lib/fetcher";
-import { User } from "@/lib/types";
+import { type User, UserStatus } from "@/lib/types";
 import useSWR, { mutate } from "swr";
+import useSWRMutation from "swr/mutation";
 import { ErrorCallout } from "@/components/ErrorCallout";
+import { HidableSection } from "@/app/admin/assistants/HidableSection";
+import BulkAdd from "@/components/admin/users/BulkAdd";
 
-const UsersTable = () => {
-  const { popup, setPopup } = usePopup();
+interface UsersResponse {
+  accepted: Array<User>;
```

Comment:
out of curiosity, any reason to use `Array<User>` vs `User[]`

Reply:
None, just a habit from previous projects.

Reply:
a072234a